src/core/server/saved_objects/migrationsv2/next.ts

```diff
@@ -58,108 +58,129 @@ export type ResponseType<ControlState extends AllActionStates> = UnwrapPromise<
+        batchSize: state.batchSize,
+        searchAfter: state.lastHitSortValue,
+      }),
     OUTDATED_DOCUMENTS_SEARCH_CLOSE_PIT: (state: OutdatedDocumentsSearchClosePit) =>
-      Actions.closePit(client, state.pitId),
+      Actions.closePit({ client, pitId: state.pitId }),
     OUTDATED_DOCUMENTS_REFRESH: (state: OutdatedDocumentsRefresh) =>
-      Actions.refreshIndex(client, state.targetIndex),
+      Actions.refreshIndex({ client, targetIndex: state.targetIndex }),
     OUTDATED_DOCUMENTS_TRANSFORM: (state: OutdatedDocumentsTransform) =>
-      Actions.transformDocs(transformRawDocs, state.outdatedDocuments),
+      Actions.transformDocs({ transformRawDocs, outdatedDocuments: state.outdatedDocuments }),
     TRANSFORMED_DOCUMENTS_BULK_INDEX: (state: TransformedDocumentsBulkIndex) =>
-      Actions.bulkOverwriteTransformedDocuments(
+      Actions.bulkOverwriteTransformedDocuments({
         client,
-        state.targetIndex,
-        state.transformedDocs,
+        index: state.targetIndex,
+        transformedDocs: state.transformedDocs,
         /**
          * Since we don't run a search against the target index, we disable "refresh" to speed up
          * the migration process.
          * Although any further step must run "refresh" for the target index
          * before we reach out to the MARK_VERSION_INDEX_READY step.
          * Right now, it's performed during OUTDATED_DOCUMENTS_REFRESH step.
          */
-        false
-      ),
+        refresh: false,
+      }),
     MARK_VERSION_INDEX_READY: (state: MarkVersionIndexReady) =>
-      Actions.updateAliases(client, state.versionIndexReadyActions.value),
+      Actions.updateAliases({ client, aliasActions: state.versionIndexReadyActions.value }),
     MARK_VERSION_INDEX_READY_CONFLICT: (state: MarkVersionIndexReadyConflict) =>
-      Actions.fetchIndices(client, [state.currentAlias, state.versionAlias]),
+      Actions.fetchIndices({ client, indicesToFetch: [state.currentAlias, state.versionAlias] }),
     LEGACY_SET_WRITE_BLOCK: (state: LegacySetWriteBlockState) =>
-      Actions.setWriteBlock(client, state.legacyIndex),
+      Actions.setWriteBlock({ client, index: state.legacyIndex }),
     LEGACY_CREATE_REINDEX_TARGET: (state: LegacyCreateReindexTargetState) =>
-      Actions.createIndex(client, state.sourceIndex.value, state.legacyReindexTargetMappings),
+      Actions.createIndex({
+        client,
+        indexName: state.sourceIndex.value,
+        mappings: state.legacyReindexTargetMappings,
+      }),
     LEGACY_REINDEX: (state: LegacyReindexState) =>
-      Actions.reindex(
+      Actions.reindex({
         client,
-        state.legacyIndex,
-        state.sourceIndex.value,
-        state.preMigrationScript,
-        false,
-        state.unusedTypesQuery
-      ),
+        sourceIndex: state.legacyIndex,
+        targetIndex: state.sourceIndex.value,
+        reindexScript: state.preMigrationScript,
+        requireAlias: false,
```

Comment:
optional: I can't find whether it's used with other values. Maybe we should remove the parameter? The same for `60s` in `waitForReindexTask` and `waitForPickupUpdatedMappingsTask`